internal/common/enums.rs

```diff
@@ -431,6 +431,18 @@ macro_rules! for_each_enums {
                 /// Scrollbar always visible
                 AlwaysOn,
             }
+
+            // This enum describes the close behaviour of `PopupWindow`
+            enum ClosePolicy {
+                /// Closes the `PopupWindow` when user clicks.
+                OnClick,
+
+                /// Closed the `PopupWindow` when user clicks outside of the popup.
+                OnClickOutside,
+
+                /// Does not close the `PopupWindow` automatically when user clicks
+                Off,
```

Comment:
I'd like to suggest that we discuss the names. My preference is a `Close` prefix for the first two policies. For the last one, the fact that "off" does not appear in the documentation suggests that it's not the correct name. I like `NoAutoClose` or (even better, with the prefix) `CloseManually`. That's the opposite of "automatic" and it works with the close prefix.